modules/installation-gcp-permissions.adoc

```diff
@@ -23,32 +24,79 @@ install {product-title}. To deploy an {product-title} cluster, the service
 account requires the following permissions. If you deploy your cluster into an existing VPC, the service account does not require certain networking permissions, which are noted in the following lists:
 
 .Required roles for the installation program
-* Compute Admin
-* Security Admin
-* Service Account Admin
-* Service Account User
-* Storage Admin
+[cols="3,7"options="header"]
+|===
+|Role|Description
+
+|Compute Admin
+(roles/compute.admin)
+|link:https://cloud.google.com/iam/docs/understanding-roles#compute-engine-roles[Compute Engine roles]
+
+|Security Admin
+(roles/iam.securityAdmin)
+|Allows the service account to create, modify, and delete firewall rules and SSL certificates within the GCP project.
+
+|Service Account Admin
+(roles/iam.serviceAccountAdmin)
+|Creates and manages service accounts. This permission is required when managing your GCP credentials using mint mode.
+
+|Service Account User
+(roles/iam.serviceAccountUser)
+|Runs operations as the service account.
+
+|Storage Admin
+(roles/storage.admin)
+|Grants full control of storage objects and buckets, and is needed to help provide persistent storage to the cluster.
+|===
+
+For more information, see link:https://cloud.google.com/iam/docs/understanding-roles[Compute Engine roles].
 
 .Required roles for creating network resources during installation
-* DNS Administrator
+[cols="3,7"options="header"]
+|===
+|Role|Description
+
+|DNS Administrator
+(roles/dns.admin)
+|Provides the service account read-write access to all Cloud DNS resources. To install {product-title}, your GCP account must have a dedicated public hosted zone in the same project that you host the cluster. This zone must be authoritative for the domain. The DNS service provides cluster DNS resolution and name lookup for external connections to the cluster. A cluster also needs this permission to add new DNS entries for the Ingress Operator through the Kubernetes cloud integration.
+|===
+
+For more information, see link:https://cloud.google.com/iam/docs/understanding-roles#dns-roles[DNS roles].
+
 
 ifdef::template[]
 .Required roles for user-provisioned GCP infrastructure
-* Deployment Manager Editor
-* Service Account Key Admin
+[cols="3,7"options="header"]
+|===
+|Role|Description
+
+|Deployment Manager Editor
+|Allows the Deployment Manager to create and manage deployments. Deployment Manager templates are provided in the {product-title} documentation as examples for deployment types.
+
+|Service Account Key Admin
+|Allows a key to be added to the service account, which is required to create a securely signed URL for the bootstrap ignition.
+
+For more information, see link:https://cloud.google.com/iam/docs/understanding-roles#deployment-manager-roles[Deployment Manager roles].
+
+|===
 endif::template[]
 
 .Optional roles
-For the cluster to create new limited credentials for its Operators, add
-the following role:
+[cols="3,7"options="header"]
+|===
+|Role|Description
+
+|Service Account Key Admin
```

Comment:
Not sure why this role (also missing the role name) is "optional". By https://deploy-preview-43941--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-mint.html#mint-mode-permissions-gcp, the default "mint" mode needs **iam.serviceAccountKeys.create** and **iam.serviceAccountKeys.delete** (both belong to this role). Could you please confirm?